datarobot_batch_scoring/network.py

```diff
@@ -33,6 +35,12 @@ def _request(self, request):
                 callback = request.hooks['response'][0]
             response = FakeResponse(400, 'No Response')
             callback(response)
+        finally:
+            request.data = None
+            prepared.data = None
+            del request
+            del prepared
+            gc.collect()
```

Comment:
How the leakage looks like? Btw - one of the things to switch to threading was that 16 number of workers allocated gigantic amount of memory. Otherwise I suppose we want to investigate it. Is it so critical? Batch-scoring wasn't priority for a long time.